CHANGELOG.md

```diff
@@ -62,6 +62,11 @@
   fine-tuning of the load-balancing between those backend services.
   [#2166](https://github.com/Kong/kubernetes-ingress-controller/issues/2166)
 
+#### Fixed
+- Fixed an issue where KIC crash loop restarts when waiting for the 
+  Kong Admin API availability
```

Comment:
I would like to expand a bit on the changelog entry for this change:

```suggestion
- Added a mechanism to retry the initial connection to the Kong
  Admin API on controller start to fix an issue with the controller
  pod could crash loop on start when waiting for Gateway
  readiness (e.g. if the Gateway is waiting for it's database to
  initialize). The new retry mechanism can be manually
  configured using the  `--kong-admin-init-retries` and
  `--kong-admin-init-retry-delay` flags.
```

Reply:
Sure thing!

Reply:
Done :)